release-notes/9.0/supported-os.json

```diff
@@ -0,0 +1,304 @@
+{
+    "channel-version": "9.0",
+    "last-updated": "2024-05-01",
+    "operating-systems": [
+        {
+            "family": "Android",
+            "distributions": [
+                {
+                    "name": "Android",
+                    "lifecycle-policy": "https://support.google.com/android",
+                    "architectures": [
+                        "x64",
+                        "arm64",
+                        "arm"
```

Comment:
x86 is also supported on Android

Reply:
Not according to https://github.com/dotnet/core/blob/main/release-notes/9.0/supported-os.md#android